src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -1234,23 +1238,23 @@ End Module").Path
             ' errors
             parsedArgs = DefaultParse({"/langVERSION", "a.vb"}, _baseDirectory)
             parsedArgs.Errors.Verify(Diagnostic(ERRID.ERR_ArgumentRequired).WithArguments("langversion", ":<number>"))
-            Assert.Equal(LanguageVersion.VisualBasic15, parsedArgs.ParseOptions.LanguageVersion)
+            Assert.Equal(LanguageVersion.Default.MapSpecifiedToEffectiveVersion(), parsedArgs.ParseOptions.LanguageVersion)
```

Comment:
Should this fix be applied to the other ones above? If not, why not? The intent isn't clear here if somebody had to do this again (or at least to me.)

Reply:
Good idea. Thanks